frontend/simple/js/transactions.js

```diff
@@ -0,0 +1,263 @@
+'use strict'
+
+import localforage from 'localforage'
+import * as Events from '../../../shared/events'
+import sbp from '../../../shared/sbp'
+
+let _steps = new WeakMap()
+let _scope = new WeakMap()
+let _cursor = new WeakMap()
+let _description = new WeakMap()
+let _isRecoverable = new WeakMap()
+let _wait = new WeakMap()
+// Why would need this?  to make our retries grow in a common sense ways
+function fibSequence (num) {
+  let output = 0
+  let sequence = [ 0, 1 ]
+  for (var i = 0; i < num; i++) {
+    sequence.push(sequence[ 0 ] + sequence[ 1 ])
+    sequence.splice(0, 1)
+    output = sequence[ 1 ]
+  }
+  return output
+}
+
+let transactionDB = localforage.createInstance({ name: 'Transactions', storeName: 'Transactions' })
```

Comment:
we need to discuss whether this is a global or per-user. i forgot what conclusion we came to last time. we need to write this conclusion down in the comments too.

Reply:
We arrived at that it was global last time. We can go back over this together.